plugins/hub/osbuild.py

```diff
@@ -69,6 +69,7 @@
             "title": "Repository options",
             "type": "object",
             "additionalProperties": False,
+            "required": ["baseurl"],
```

Comment:
I'm confused, Cloud API doesn't require `baseurl`, does it?

Reply:
I'll add my explanation from here to the commit message...

Reply:
I amended the commit message.